### PR TITLE
DES-1721 - Add informative message inside trash directories.

### PR DESCRIPTION
**Pull request by duckonomy**
## PR Status: ##

* [X] Ready.
* [ ] Work in Progress.
* [ ] Hold.

## Related Jira tickets: ##

* [DES-1721](https://jira.tacc.utexas.edu/browse/DES-1721)

## Summary of Changes: ##
* Added the message, "Trashed items will be kept a maximum of 90 days", between the breadcrumb and file-listing of My Data. 
* Added a basic test to see if the message appears in specified paths.

## Testing Steps: ##
1. Run `npm run test` and see if tests pass.
2. Go to "My Data" from "Data Depot"
3. If there is no ".Trash" create a folder and move it to trash.
4. Go to the ".Trash" directory or any of its subdirectories.
5. Confirm that the message appears.
6. Go to "My Projects" from "Data Depot"
7. Create a Project if there is none.
8. If there is no ".Trash" create a folder and move it to trash.
4. Go to the ".Trash" directory or any of its subdirectories.
5. Confirm that the message appears.

## UI Photos:
Screenshot from 2020-10-21 10-29-31
Screenshot from 2020-10-21 10-29-51
Screenshot from 2020-10-21 10-30-12
Screenshot from 2020-10-21 10-30-26

## Notes: ##
I wasn't sure if it was ok to put this directly inside `<caption>` and not through the `message` api. But this this seemed like it had to be a separate message.

### CODE REVIEW

**Review by nathanfranklin — COMMENTED** [review]
Great work!   I just have one thing to consider otherwise looks good to me 👍 

Excellent tests 💯 🏆

File: designsafe/static/scripts/data-depot/components/data-depot-listing/data-depot-listing.component.js [hunk 1]
```diff
@@ -149,6 +149,15 @@ class FilesListingCtrl {
                 return 'fa-file-o';
         }
     };
+
+    isInTrash() {
+        if (this.listing.params.path) {
+            let filePath = this.listing.params.path.split('/')
+            if (filePath[1] == '.Trash' || filePath[0] == '.Trash')
```
[the remark this answers is not in the document]
Comment (duckonomy) [reply designsafe/static/scripts/data-depot/components/data-depot-listing/data-depot-listing.component.js:156]: That sounds good! I'll try doing that.

File: designsafe/static/scripts/data-depot/components/data-depot-listing/data-depot-listing.component.js [hunk 2]
```diff
@@ -149,6 +149,18 @@ class FilesListingCtrl {
...
     };
+
+    isInTrash() {
+        if (this.listing.params.path && this.listing.params.system) {
+            let filePath = this.listing.params.path.split('/');
+            if (this.listing.params.system == 'designsafe.storage.default' &&
+                filePath[1] == '.Trash')
+                return true;
+            else if (filePath[0] == '.Trash')
+                return true;
```
Comment (duckonomy) [inline designsafe/static/scripts/data-depot/components/data-depot-listing/data-depot-listing.component.js:160]: @nathanfranklin Would this be better?

File: designsafe/static/scripts/data-depot/components/data-depot-listing/data-depot-listing.component.js [hunk 3]
```diff
@@ -149,6 +149,18 @@ class FilesListingCtrl {
...
+        }
+        return false;
```
Comment (duckonomy) [inline designsafe/static/scripts/data-depot/components/data-depot-listing/data-depot-listing.component.js:162]: @nathanfranklin Would this be better?

**Review by rstijerina — APPROVED** [review]
Nice work!

**Comment by duckonomy** [comment]
Thank you!

**Review by nathanfranklin — APPROVED** [review]
Excellent 👍

**Pull request merged**